library/core/src/num/f32.rs

```diff
@@ -977,17 +985,25 @@ impl f32 {
     /// operand is conserved; see the [specification of NaN bit patterns](f32#nan-bit-patterns) for more info.
     #[must_use = "this returns the result of the comparison, without modifying either input"]
     #[unstable(feature = "float_minimum_maximum", issue = "91079")]
+    #[rustc_const_unstable(feature = "float_minimum_maximum", issue = "91079")]
```

Comment:
Might be worth adding mode fallback bodies to avoid this attribute? (No strong opinion either way)